includes/EDD_SL_Plugin_Updater.php

```diff
@@ -295,6 +295,13 @@ private function api_request( $_action, $_data ) {
             'url'        => home_url()
         );
 
+	$cache_key = md5( 'edd_plugin_' . sanitize_key( $api_params['item_name'] . $api_params['item_id'] ) . '_' . $api_params['edd_action'] );
```

Comment:
`item_name` and `item_id` are not always included. How about passing in the license key instead of item_id and item_name.